File: api-gin/src/routes/pedido-routes.go
package routes

import (
	"github.com/gin-gonic/gin"
	controller "tcc-api-gin/src/controllers"
	middleware "tcc-api-gin/src/middlewares"
)

func PedidoRoutes(router *gin.RouterGroup) {
	pedido := router.Group("/pedidos")
	{
		pedido.POST("/", middleware.IsAuthenticated, controller.CreatePedido)
	}
}

File: api-express/src/routes/pedido-routes.js
import { Router } from "express";
import PedidoController from "../controllers/pedido-controller.js";
import isAuthenticated from "../middlewares/is-authenticated.js";

const pedidoRoutes = Router();

pedidoRoutes.post('/', isAuthenticated, PedidoController.create);

export default pedidoRoutes;